pkg/api/aim/helpers.go

```diff
@@ -0,0 +1,271 @@
+package aim
+
+import (
+	"bufio"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/rotisserie/eris"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/exp/slices"
+
+	"github.com/G-Research/fasttrackml/pkg/api/aim/encoding"
+	"github.com/G-Research/fasttrackml/pkg/database"
+)
+
+// FormatRunsSearchResponseAsCSV formats and sends Runs search response as a CSV file.
+//
+//nolint:gocyclo
+func FormatRunsSearchResponseAsCSV(ctx *fiber.Ctx, runs []database.Run, excludeTraces, excludeParams bool) {
```

Comment:
maybe these two functions could have shorter names? "RunsAsCsv", "RunsAsStream" for instance?

Reply:
maybe these two functions could have shorter names? "RunsAsCsv", "RunsAsStream" for instance? - yyyep, Ill make it a bit shorter.

Reply:
```
also, I wonder if they might be better as methods of a new struct like response.SearchRuns 
```
- Im not sure about that, since `response.SearchRuns` is from `mlflow` but in general it should be a bit different I agree. Let's think about that when we will rewrite `AIM`.

Reply:
we do have other `response` structs in Aim side already, such as `response.GetRunMetrics`, etc, so I don't think we need to wait for big rewrite

Reply:
the strange moment here, that we don't need a `struct`, because there is no real `response` object for such endpoint.